kolibri/plugins/management/assets/src/vue/user-page/user-create-modal.vue

```diff
@@ -138,7 +138,7 @@
           // using promise to ensure that the user is created before closing
           this.createUser(newUser).then(
             () => {
-              this.close();
+              this.$emit('close', this.username);
```

Comment:
can use the state-modal-control pattern like this:
https://github.com/learningequality/kolibri/pull/992/files#diff-097c60a1df0c3c5068de7ce8b5575a9aL80